### ISSUE TITLE
No description about forcing zero output for mask in Bidirectional

### ISSUE THREAD

**Issue by cosmoquester** [tensorflow/tensorflow#49738]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

Please provide a link to the documentation entry: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Bidirectional

### Clear description

```python
import tensorflow as tf

lstm = tf.keras.layers.LSTM(4, return_sequences=True)
bi = tf.keras.layers.Bidirectional(lstm)

x = tf.random.normal([1,4,16])
mask = tf.constant([[True, True, True, False]])

print(lstm(x, mask=mask))
print(bi(x, mask=mask))
```

```
tf.Tensor(
[[[ 0.01245601  0.38689056  0.01844893 -0.0718843 ]
  [ 0.14610071  0.23905458  0.39626616 -0.17714866]
  [-0.00543382 -0.06880241  0.04203304 -0.08996341]
  [-0.00543382 -0.06880241  0.04203304 -0.08996341]]], shape=(1, 4, 4), dtype=float32)
tf.Tensor(
[[[-0.24271122  0.05120631 -0.06832076 -0.5101022   0.3812662
    0.44380718 -0.07919203  0.07195219]
  [-0.16884208  0.18173794 -0.0029141  -0.13847476  0.08567893
    0.2971174   0.15979137  0.01258926]
  [-0.11614764  0.17977336 -0.20486659 -0.0677676   0.40112934
    0.27802816 -0.20526664  0.1625048 ]
  [ 0.          0.          0.          0.          0.
    0.          0.          0.        ]]], shape=(1, 4, 8), dtype=float32)
```

https://github.com/tensorflow/tensorflow/blob/5dcfc51118817f27fad5246812d83e5dccdc5f72/tensorflow/python/keras/layers/wrappers.py#L492-L498
When we use `tf.keras.layers.Bidirectional` with `return_sequences=True`, The output for masked sequences become zero because **force_zero_output_for_mask** internally. However, there is no description that the masked timestep should be zero with `return_sequences=True` in Bidirectional layer documentation.
Without `Bidirectional`, `zero_output_for_mask` is false by default. Many of people hardly expect the output of masked timestep is zero. So I think this is very confused and easy to misunderstand.

I think the fact that it forces zero oputput for mask internally should be added to the Bidirectional document.

### Submit a pull request?

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs,
docs API guide: https://www.tensorflow.org/community/contribute/docs_ref and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

Yes I may be able to add document about this.

**Comment by cosmoquester** [comment]
Is it ok to close issue because the PR merged?

**Comment by rmothukuru** [comment]
@cosmoquester,
Yes please. Thank you for your contribution.
